pkg/ingress/cert_discovery.go

```diff
@@ -102,7 +106,10 @@ func (d *acmCertDiscovery) loadDomainsForAllCertificates(ctx context.Context) (m
 		if err != nil {
 			return nil, err
 		}
-		domainsByCertARN[certARN] = certDomains
+		if len(certDomains.List()) > 0 {
```

Comment:
you can directly use `len(certDomains)` which is more efficient than convert it into list.

Reply:
thx for the hint, I updated it